src/Plugin/EdgeKeyTypeInterface.php

```diff
@@ -90,32 +92,38 @@ public function getPassword(KeyInterface $key): string;
    *
    * @param \Drupal\key\KeyInterface $key
    *   The key entity.
+   * @param bool $safeReturn
+   *   If TRUE then the fallback value is the default value provided by the SDK.
    *
-   * @return string
+   * @return string|null
    *   The authorization server.
    */
-  public function getAuthorizationServer(KeyInterface $key): string;
+  public function getAuthorizationServer(KeyInterface $key, bool $safeReturn = FALSE): ?string;
 
   /**
    * Gets the client ID.
    *
    * @param \Drupal\key\KeyInterface $key
    *   The key entity.
+   * @param bool $safeReturn
+   *   If TRUE then the fallback value is the default value provided by the SDK.
    *
-   * @return string
+   * @return string|null
    *   The client ID.
    */
-  public function getClientId(KeyInterface $key): string;
+  public function getClientId(KeyInterface $key, bool $safeReturn = FALSE): ?string;
 
   /**
    * Gets the client secret.
    *
    * @param \Drupal\key\KeyInterface $key
    *   The key entity.
+   * @param bool $safeReturn
+   *   If TRUE then the fallback value is the default value provided by the SDK.
```

Comment:
Please also describes what is the default behavior, what FALSE means. Start your sentence with `(optional) ...` because this parameter has a default value.